front/app/containers/Authentication/steps/SSOVerification/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+
+import { SSOProvider } from 'api/authentication/singleSignOn';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import TextButton from '../_components/TextButton';
+import AuthProviderButton from '../AuthProviders/AuthProviderButton';
+import authProviderMessages from '../AuthProviders/messages';
+
+import messages from './messages';
+
+interface Props {
+  onClickSSO: (ssoProvider: SSOProvider) => void;
+  onClickLogin: () => void;
+}
+
+const SSOVerification = ({ onClickSSO, onClickLogin }: Props) => {
+  const fakeSsoEnabled = useFeatureFlag({ name: 'fake_sso' });
```

Comment:
Assume the others aren't here yet because we'll add them when we enable them?

Reply:
Indeed :+1: